src/node/index.ts

```diff
@@ -517,6 +530,30 @@ export default function PluginInspect(options: Options = {}): Plugin {
     return targetDir
   }
 
+  function createInternalServer(staticPath: string) {
+    const server = createServer()
```

Comment:
Why do we need to create a server here?

Reply:
I want to create a static server hosting `.vite-inspect`